Project.toml

```diff
@@ -3,20 +3,20 @@ uuid = "b3cc710f-9c33-5bdb-a03d-a94903873e97"
 version = "4.5.1-DEV"
 
 [deps]
-Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 CSTParser = "00ebfdb7-1f24-5e51-bd34-a7502290713f"
+Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 SymbolServer = "cf896787-08d5-524d-9de7-132aaa0cb996"
 
+[compat]
+CSTParser = "2.3.1"
+SymbolServer = "5.1.1-DEV"
```

Comment:
This file code jumbled a bit when I updated the environment, but this line should be the only real change